test/controllers/actions/AuthActionSpec.scala

```diff
@@ -45,7 +45,7 @@ class AuthActionSpec extends SpecBase {
 
     lazy val bodyParsers: BodyParsers.Default = injector.instanceOf[BodyParsers.Default]
     lazy val unauthorisedView: UnauthorisedView = injector.instanceOf[views.html.errors.UnauthorisedView]
-    lazy val config = new MockAppConfig(List.empty, false, false, false)
+    lazy val config = new MockAppConfig(List.empty, false, false, false, true)
```

Comment:
Can you name the arguments for MockAppConfig in this file please? Also include the ones that are not for otherItem too for readability.